bot/logger/logger.py

```diff
@@ -0,0 +1,60 @@
+"""Logger Module. Configures the logger for the discord library and the logger for custom code.
```

Comment:
"""Module for logging.

Configures and sets up loggers for the library discord.py and our own code.
"""